package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hpke",
   "description": "hybrid public key encryption",
-  "version": "0.3.0",
+  "version": "0.4.0",
```

Comment:
Does the version in `Cargo.toml` need to be bumped in sync with this?

Reply:
Previous to #18 there were no changes in the rust interface so it seemed strange to bump the version (and there's no reason the two versions need to be kept in sync) but now there is cause for a minor version bump in both

Reply:
OK. I think we should bump the crate to 0.4.0 in this PR, and once we merge it, I'll release 0.4.0 to crates.io.

Reply:
#cd56647